icons/ktm.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>KTM icon</title><path d="M0 15.735h3.3539l.843-2.0605 1.5507 2.0606h7.2245l2.234-2.0814-.3722 2.0814h2.8309l2.335-2.0597-.3197 2.0597h3.0522L24 9.9894h-3.068L18.4458 12.18l.4802-2.1907h-2.9417l-3.209 3.2161 1.3414-3.9371h4.9077l.2248-1.0035H6.3815L6.003 9.2684h4.7322l-1.9939 5.054-1.5723-2.066 2.717-2.267H7.612l-2.7876 2.229.9385-2.229H2.44L0 15.735" /></svg>
```

Comment:
My vote also goes to the precision 3 version, could you revert the change @irgusite (or use the suggestion below)

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>KTM icon</title><path d="M0 15.735h3.354l.843-2.06 1.55 2.06h7.225l2.234-2.081-.372 2.081h2.83L20 13.675l-.32 2.06h3.052L24 9.99h-3.068l-2.486 2.191.48-2.19h-2.942l-3.209 3.216 1.342-3.938h4.907l.225-1.003H6.381l-.378 1.003h4.732l-1.994 5.054-1.572-2.066L9.886 9.99H7.612l-2.787 2.23.938-2.23H2.44L0 15.735"/></svg>
```